.github/workflows/palette.yml

```diff
@@ -0,0 +1,28 @@
+name: Palette CI
+
+on:
+  push:
+    paths:
+      - 'palette/**'
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./palette
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
now that I'm searching, 16 has already met eol.
https://vercel.com/changelog/node-js-16-deprecation

Have you tried this with v18? (FYI, field-plugin repo is using 18.16.0) But if it changes way too much, let's postpone the version upgrade to later.

Also, it'd be nice to have `.nvmrc` file within `palette/` folder to specify whoever works on it.

Reply:
Indeed, the palette is only working 'till 16. That's also why I skipped adding it as a workspace and added the `working-directory` as the other packages used higher node versions.

I also tried to upgrade the packages to some extent, but there were a lot of changes needed for this small PR that I thought was out of scope. I will include the `.npmrc` thanks for pointing it out.

What shall I do about the migration then?

Reply:
Ah I think you mentioned this yesterday, and I think I didn't fully understand it back then. Yes, it makes sense. And let's keep it to 16 for now, and only include `.nvmrc` (you mean `.nvmrc`, but not `.npmrc`, right?)

Could you create a ticket on our backlog to migrate this to node v18? We can tackle it later.